simplivity/resources/backups.py

```diff
@@ -184,3 +190,20 @@ def restore(self, restore_original, virtual_machine_name=None, datastore=None, t
         affected_object = self._client.do_post(resource_uri, data, timeout, None, flags)[0]
         virtual_machines_obj = virtual_machines.VirtualMachines(self._connection)
         return virtual_machines_obj.get_by_id(affected_object["object_id"])
+
+    def rename(self, backup_name, timeout=-1):
```

Comment:
Can we use new_name as method argument instead of backup_name?

Reply:
Updated as per the suggestion